irohad/wsv_checker/.clang-format

```diff
@@ -0,0 +1,16 @@
+BasedOnStyle: Mozilla
+#BasedOnStyle: Microsoft
+#BasedOnStyle: Google
+#BasedOnStyle: Chromium
+#BasedOnStyle: WebKit
+#BasedOnStyle: LLVM
+IndentWidth: 3
+ColumnLimit: 120
+#Cpp11BracedListStyle: false
+Cpp11BracedListStyle: true
```

Comment:
```wsv_checker``` formatting now differs from the other project. I don't understand why.

Reply:
removed